src/components/Homepage/LendByCategory/LoanCategoriesSection.vue

```diff
@@ -0,0 +1,354 @@
+<template>
+	<section class="loan-categories">
+		<div class="row">
+			<div class="columns">
+				<h2 class="large-text text-center">
+					Kiva makes it easy to support causes you care about.
+				</h2>
+
+				<div>
```

Comment:
Haha, we're stepping on each others toes already :). Do you mind making the component from this div down, and keeping the section + h2 tags as part of LendByCategoryHomepage?

Reply:
@ryan-ludwig done

Reply:
@eddieferrer Thank you!

Reply:
@ryan-ludwig added that refactor here